### DISCUSSION TITLE
Czech translation devilutionX

### DISCUSSION THREAD

**Discussion by fanstar82** [diasurgical/DevilutionX#4143] [Q&A]
Hi,
I have identified an issue with the czech translation. The translation of dialogues with village characters (Cain and the others) does not work - it is shown in eglish. The all other descriptions are properly shown in czech. Could you please repair it? Thanks.

**Comment by AJenbo** [comment]
It's not an issue. It has simply not been fully translated. I don't speak Czech, if you have time you are welcome to help finish the Czech translation. The translation is 87% complete and only 248 lines needs to be translated.
